service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandlerTest.java

```diff
@@ -0,0 +1,320 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.events.FurtherEvidenceUploadedEvent;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.SupportingEvidenceBundle;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.FurtherEvidenceNotificationService;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ExtendWith(MockitoExtension.class)
+class FurtherEvidenceUploadedEventHandlerTest {
+    private static final String CASE_REFERENCE = "12345";
```

Comment:
private static final String CASE_REFERENCE = CASE_ID.toString()

or maybe you dont need it at all

Reply:
I've removed this usage but left the tests unchanged.